doc/guides/west/config.rst

```diff
@@ -150,3 +150,29 @@ commands.
        environment overrides the value of the ``zephyr.base`` configuration
        option. If set to ``"configfile"``, the configuration option wins
        instead.
+
+Zephyr Extension Commands Configuration Options
+-----------------------------------------------
+
+The following table documents configuration options supported by zephyr's
```

Comment:
``Zephyr`` (upper case)

Reply:
Actually I intentionally used lower-case because I am referring to the repository, since extension commands are per-repository.